backend/src/controllers/student.ts

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable @typescript-eslint/no-misused-promises */
+/**
+ * Functions that process task route requests.
+ */
+
+import { RequestHandler } from "express";
+import { validationResult } from "express-validator";
+
+import { CustomError } from "../errors/errors";
+import { errorHandler } from "../errors/handler";
+import StudentModel from "../models/student";
+
+export const createStudent: RequestHandler = async (req, res, next) => {
+  type contact = {
+    lastName: string;
+    firstName: string;
+    email: string;
+    phoneNumber: string;
+  };
+
+  type typedModel = {
```

Comment:
We should move these types outside of the route so that it can be accessed by future routes.